### PR TITLE
fix: No unsaved change shown when adding fragment to extension point

### PR DESCRIPTION
**Pull request by nikmace**
Fix for #1526.

- Changes prepareChangeType method to retrieve correctly the information for `addXMLAtExtensionPoint` and `setFlexExtensionPointEnabled` changes.

### CODE REVIEW

File: packages/preview-middleware-client/src/cpe/changes/service.ts [hunk 1]
```diff
@@ -310,6 +310,14 @@ export class ChangeService {
...
             };
+        } else if (changeType === 'addXMLAtExtensionPoint') {
+            result = {
+                type: 'pending',
+                controlId: selectorId,
+                changeType,
+                isActive: index >= inactiveCommandCount,
+                controlName: command.getSelector().name ?? ''
+            };
         } else {
```
Comment (GDamyanov) [inline packages/preview-middleware-client/src/cpe/changes/service.ts:321]: Here we have duplicated code, so can we remove this else if statement and just in the result controlName put a ternary operator something like this: `result = {
                type: 'pending',
                controlId: selectorId,
                changeType,
                isActive: index >= inactiveCommandCount,
                controlName:
                    changeType === 'addXMLAtExtensionPoint'
                        ? command.getSelector().name ?? ''
                        : command.getElement().getMetadata().getName().split('.').pop() ?? ''
            };`

Comment (nikmace) [reply packages/preview-middleware-client/src/cpe/changes/service.ts]: Changed in: [9545333](https://github.com/SAP/open-ux-tools/pull/1527/commits/9545333ef711c253902fbde4a44b999f8790166a)

**Review by GDamyanov — APPROVED** [review]
Code looks good
Good test coverage
Comments addressed
Manual testing done

**Review by zdravko-georgiev — APPROVED** [review]
Code looks good
Good test coverage
Manual testing ✅

**Pull request merged**